### PR TITLE
Fixed several unused variables issues

### PR DESCRIPTION
**Pull request by JanGerritsen**
Removed several unused local variables

### CODE REVIEW

**Comment by JanGerritsen** [comment]
Might there be a problem in the CI Build? All Java Versions except Java 9 ran successfully, and the test for Java 9 just has stopped.
Is there a way to retrigger the Travis CI build?

**Comment by desyncr** [comment]
@JanGerritsen Hey! Thanks for the work! Regarding the build failure I think you can force a re-build by pushing an empty commit: https://stackoverflow.com/a/31694534

Otherwise someone with Write access to the repo could trigger the build using Travis UI, @ArneBab ?

File: src/freenet/node/PeerManager.java
Comment (Bombe) [inline src/freenet/node/PeerManager.java:1794]: The second `Disconnecting` seems to be mislabeled.

**Review by Bombe — CHANGES_REQUESTED** [review]

File: src/freenet/node/PeerManager.java
Comment (JanGerritsen) [reply src/freenet/node/PeerManager.java]: Yes, you are right. Fixed it.

**Comment by ArneBab** [comment]
@Bombe is this good to go?

**Comment by Bombe** [comment]
As soon as somebody confirms that all removed method calls do not have any side effects, sure. :)

**Comment by desyncr** [comment]
@Bombe Do you mean these ones:

- https://github.com/freenet/fred/pull/707/files#diff-5ee75719a7e50239b28aa9afc54916bccf13373d2b659220627834d07911dfaaL103
- https://github.com/freenet/fred/pull/707/files#diff-65af551d910019a886b79d1a718d2086f20f69c9b63a4290a9fc4b74f7472af5L2814

I believe the rest are unused variables being removed.

**Comment by ArneBab** [comment]
Does that mean it’s good to go?

**Comment by desyncr** [comment]
@ArneBab If the problem is with the removed methods specifically then we can leave them until we're certain there are no side effect, while removing the unused variables alone. Does that sound good to you @Bombe @JanGerritsen ?

**Comment by Bombe** [comment]
I checked the methods and them seem to be free enough of side-effects to simply remove them.

**Comment by desyncr** [comment]
Right on! Anything else left to do here?

**Comment by desyncr** [comment]
@Bombe Are we good to go?

**Review by desyncr — APPROVED** [review]

**Comment by Bombe** [comment]
Sure.

**Comment by desyncr** [comment]
@Bombe Great! I guess we'll need an approval from you @Bombe before it can be merged.

**Review by Bombe — APPROVED** [review]

**Comment by Bombe** [comment]
Oops, sorry for the delay, didn’t realize I had to push the button. 😄

**Comment by desyncr** [comment]
@ArneBab Should we be ready to go!

**Comment by ArneBab** [comment]
Thank you for your changes and thank you for the review!

**Pull request merged**
